corehq/apps/linked_domain/tests/test_update_tableau_visualization.py

```diff
@@ -0,0 +1,45 @@
+from corehq.apps.reports.models import TableauVisualization, TableauServer
+from corehq.apps.linked_domain.local_accessors import get_tableau_visualizaton
+from corehq.apps.linked_domain.tests.test_linked_apps import BaseLinkedAppsTest
+from corehq.apps.linked_domain.updates import update_tableau_visualization
+
+
+class TestUpdateTableauVisualization(BaseLinkedAppsTest):
+    def setUp(self):
+        self.server = TableauServer(domain=self.domain,
+        server_type='server',
+        server_name='server name',
+        validate_hostname='host name',
+        target_site='target site',
+        domain_username='username',
+        allow_domain_username_override=True)
+        self.server.save()
+        self.tableau_visualization_setup = TableauVisualization(domain=self.domain,
+        server=self.server, view_url='url')
+        self.tableau_visualization_setup.save()
+    def tearDown(self):
+        self.server.delete()
+        # self.tableau_visualization_setup.delete()
```

Comment:
is this meant to be commented out?

Reply:
Yeah, it was meant to be removed. My logic was that the cascade setting on the visualization model would remove the visualization when the server is deleted. It seemed to work when tested. I'll remove it now.